app.R

```diff
@@ -0,0 +1,843 @@
+    ),
+    ### Sidebar ----
+    dashboardSidebar(
+      width = 250,
+      sidebarMenu(
+        id = "pages",
+        menuItem("Overview", tabName = "overview",icon = icon("dashboard")),
+        menuItem("Survival Rate", tabName = "survivalRate", icon = icon("wpexplorer")),
+        menuItem("Static Pop. Pyramids", tabName = "staticPry", icon = icon("wpexplorer")),
+        menuItem("Time & Pop. Pyramids", tabName = "timePry", icon = icon("wpexplorer")),
+        menuItem("Fecundity Rate", tabName = "fecundityRate", icon = icon("wpexplorer")),
+        menuItem("References", tabName = "references", icon = icon("leanpub"))
+      ),
+      tags$div(
+        class = "sidebar-logo",
+        boastUtils::sidebarFooter()
+      )
+    ),
+    ### Body ----
+    dashboardBody(
+      tabItems(
+        #### Overview Page ----
+        tabItem(
+          tabName = "overview",
+          h1("Life Tables"),
+          p("In this app, you will explore various ways to visualize life tables
+            for 3 different countries--China, the United Kingdom, and the United
+            States of America. You'll be able to look at the several different
+            metrics associated with life tables (e.g., survival and fecundity
+            rates) by country and by sex."),
+          h2("Instructions"),
+          p("Use the left sidebar menu to explore the following life tables."),
+          tags$ul(
+            tags$li(tags$strong("Survival Rate:"), " Click the country and sex
+                    combination you preferred to compare, and view the distinct
+                    Survival Rate for each country and sex."),
+            tags$li(tags$strong("Static Population Pyramids:"), " View and compare
+                    Population Pyramids for different countries, as well as the
+                    combination of sex and country."),
+            tags$li(tags$strong("Population Pyramids Through Time:"), " Click the start
+                    button to watch the population change by year in United
+                    States and United Kingdom."),
+            tags$li(tags$strong("Fecundity Rate:"), " Click the country you
+                    want to explore and then view the different Fecundity Rate
+                    (per 1,000 women).")
+          ),
+          div(
+            style = "text-align:center;",
+            bsButton(
+              inputId = "go",
+              label = "Survival Rate",
+              icon = icon("wpexplorer"),
+              size = "large",
+              style = "default"
+            )
+          ),
+          br(),
+          br(),
+          h2("Acknowledgements"),
+          p("This app was developed and coded by Yuqing Lei in 2019 with the
+            support of funding provided by Dr. Stephen Schaeffer. The app was
+            updated in 2021 by Dr. Neil J. Hatfield.",
+            br(),
+            br(),
+            br(),
+            div(class = "updated", "Last Update: 7/7/2021 by NJH.")
```

Comment:
For some reason, when I scroll down after this there is a large white space.

Reply:
I don't know what you mean by this.

Reply:
It's no longer there, so no worries! Yesterday the page kept scrolling way past the updated statement so there was just a bunch of empty space before I could stop scrolling even though there was nothing after this statement on the overview page.